sql/src/planner.rs

```diff
@@ -380,6 +387,15 @@ impl<'a, P: MetaProvider> PlannerDelegate<'a, P> {
 
         // Timestamp column must be provided.
         let timestamp_name = timestamp_name.context(RequireTimestamp)?;
+        // The timestamp key column must not be a Tag column
+        if let Some(timestamp_column) = name_column_map.get(&timestamp_name as &str) {
```

Comment:
I find the key type of name_column_map is `&str`, and it will be better if we can put it as:
```suggestion
        if let Some(timestamp_column) = name_column_map.get(&timestamp_name.as_str()) {
```

Reply:
OK